Content.Server/Flash/FlashSystem.cs

```diff
@@ -139,6 +140,11 @@ private bool UseFlash(EntityUid uid, FlashComponent comp, EntityUid user)
             _stun.TrySlowdown(target, TimeSpan.FromSeconds(flashDuration/1000f), true,
                 slowTo, slowTo);
 
+            if (stunTime != null)
+            {
+                _stun.TryParalyze(target, TimeSpan.FromSeconds(stunTime.Value/1000f), true);
+            }
+
```

Comment:
Should this be a separate component instead? i.e. StunAfterFlashComponent then subscribe to AfterFlashEvent.